### ISSUE TITLE
Error Parameter "Value"

### ISSUE THREAD

**Issue by ElricvM06** [FPV-Combat/FPVC-GUI#13]
![WhatsApp Bild 2024-08-29 um 08 32 14_5fd40e5a](https://github.com/user-attachments/assets/50fc4b88-4398-403c-9f18-a40d4616fa8d)
Tried to configure HW26 with FW 2.8.CAMP with Configurator v1.1. Error Message rises at reading and writing.

**Comment by chaos5235** [comment]
yeah this is a known issue at the moment - will be fixed in next version
